src/editor/adapters/change-signature-webview/change-signature.html

```diff
@@ -0,0 +1,137 @@
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+      }
+
+      td,
+      th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+      }
+
+      th:last-child {
+        border-top-color: transparent;
+        border-right-color: transparent;
+      }
+
+      .up,
+      .down {
+        cursor: pointer;
+        display: inline-block;
+        width: 8px;
+        margin: 0 0.7rem;
+        font-size: 1.2rem;
+      }
+
+      .up:after {
+        content: "▲";
+      }
+
+      .up:hover:after {
+        color: #625e5e;
+      }
+
+      .down:after {
+        content: "▼";
+      }
+
+      .down:hover:after {
+        color: #625e5e;
+      }
+
+      button {
+        border: 1px solid transparent;
+        border-radius: 5px;
+        line-height: 1.25rem;
+        outline: none;
+        text-align: center;
+        white-space: nowrap;
+        display: inline-block;
+        text-decoration: none;
+        background: #1a85ff;
+        padding: 4px;
+        color: white;
+        font-size: 14px;
+      }
+
+      button:hover {
+        cursor: pointer;
+        color: white;
+      }
+    </style>
+  </head>
+
+  <body>
+    <h4>Parameters</h4>
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th></th>
+        </tr>
+      </thead>
+
+      <tbody id="params">
+        {{tableContent}}
+      </tbody>
+      <tfoot>
+        <tr>
+          <td colspan="2" style="text-align: center">
+            <button id="confirm">Confirm</button>
+          </td>
+        </tr>
+      </tfoot>
+    </table>
+
+    <div class="btn-wrapper"></div>
+
+    <script>
+      const vscode = acquireVsCodeApi();
+      const startValues = document.querySelectorAll("#params .params-name");
+
+      function moveUp(element) {
+        if (element.previousElementSibling)
+          element.parentNode.insertBefore(
+            element,
+            element.previousElementSibling
+          );
+      }
+
+      function moveDown(element) {
+        if (element.nextElementSibling)
+          element.parentNode.insertBefore(element.nextElementSibling, element);
+      }
+
+      document.querySelector("#params").addEventListener("click", function (e) {
+        if (e.target.className === "down")
+          moveDown(e.target.parentNode.parentNode);
+        else if (e.target.className === "up")
+          moveUp(e.target.parentNode.parentNode);
+      });
+
+      document.querySelector("#confirm").addEventListener("click", () => {
+        const tdsElements = document.querySelectorAll("#params .params-name");
+        const tds = Array.from(tdsElements);
+
+        const items = Array.from(startValues).map((item, index) => {
+          const endAt = tds.findIndex((td) => td === item);
+
+          return {
+            label: item.innerHTML,
+            startAt: index,
+            endAt: endAt
+          };
+        });
+
+        vscode.postMessage({
```

Comment:
💡 Interesting to see how the template can communicate back & forth with VS Code.